Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -293,6 +295,8 @@ export async function getProvisioningProfileName(provProfilePath: string) {
 export async function getProvisioningProfileType(provProfilePath: string) {
```

Comment:
Since this method got complex, looks like it will be better to separate on the app type - iOS vs macOS. If I follow correctly, iOS can have - enterprise, app-store, ad-hoc and development. For macOS - developer-id, app-store and development. We should add more comments at least.

Reply:
I split this into two different functions.

Reply:
Definitely more readable though there is more code duplication now.